browser-test/src/applicant/questions/file.test.ts

```diff
@@ -445,6 +445,29 @@ test.describe('file upload applicant flow', () => {
       )
     })
 
+    test('can upload file with very long file name', async ({
```

Comment:
(blocking) We don't need a separate test just for this; it's too fine grained for the browser tests. Just add the long file name to whichever test is the primary success path of multi-file test.

Reply:
Done.